ushahidi/platform-client
# Installer web checks

This pull request makes the following changes:

- Adds FE-verifier with Angular, go to /verifier to see.
- Moves all verifier-tests to a specific file, that can be used both for the gulp-verifiers and browser-verifiers.
-Refactors the gulp-verifier to use the specific verifier-test-file instead
-Adds missing checks for dbConnection and env in the api (needs branch db-installer-helper in the api)

Testing checklist:
- in the cli, run "gulp verify"
- [x] A list with checks and statuses is listed
- run gulp dev:verifier
- go to /verifier
- [x] A list with checks and statuses is listed
- Destroy something, like stopping the database or remove the BACKEND_URL from the .env-file
- [x] Reload, the browser, it should show the checks, with errors
- Go to the cli again, run gulp verify
- [ ] It should list the same errors as in the browser

 I certify that I ran my checklist
Ping @ushahidi/platform
